mvsim_node_src/include/mvsim/mvsim_node_core.h

```diff
@@ -91,8 +146,8 @@ class MVSimNode
 
 	/// The mvsim library simulated world (includes everything: vehicles,
 	/// obstacles, etc.)
-	std::shared_ptr<mvsim::World> mvsim_world_ =
-		std::make_shared<mvsim::World>();
+	mvsim_node::shared_ptr<mvsim::World> mvsim_world_ =
```

Comment:
We should move to clang-format `ColumnLimit: 100` as so many modern projects... 

If you feel like doing it in this same PR, go on (and feel free to rebase/force push etc. to save commits), otherwise it will be done another day.